bin/run.sh

```diff
@@ -72,8 +76,17 @@ cp "${manifest_file}" "${manifest_file_bak}"
 cp "${gleam_file}" "${gleam_file_bak}"
 rm -fr "$solution_dir"/build
 cp -r "$root_dir"/packages/build "$solution_dir"/build
-cp "$root_dir"/packages/manifest.toml "${manifest_file}"
-cat "$root_dir"/packages/gleam.toml | sed "s/name = \".*\"/name = \"$underscore_slug\"/" > "${gleam_file}"
+if $local; then
+  cat "$root_dir"/packages/manifest.toml |
+    sed "s|{ name = \"exercism_test_runner\", version = \"\([0-9.]*\)\", build_tools = \[\"gleam\"\], requirements = \[\(.*\)\].*}|{ name = \"exercism_test_runner\", version = \"\1\", build_tools = [\"gleam\"], requirements = [\2], source = \"local\", path = \"${root_dir}/runner\" }|" |
```

Comment:
You can use single quotes on the outside then you don't need to escape `"` chars.

```suggestion
    sed 's|{ name = "exercism_test_runner", version = "\([0-9.]*\)", build_tools = \["gleam"\], requirements = \[\(.*\)\].*}|{ name = "exercism_test_runner", version = "\1", build_tools = ["gleam"], requirements = [\2], source = "local", path = "${root_dir}/runner" }|" |
```

You don't need `cat`; `sed` can read from files, too.

Would a `toml` tool make more sense here than `sed`?

Reply:
With single quotes, I can't use the `${root_dir}` inside the string. 

As for toml manipulation tools, I haven't used any so far, but my guess would also be that a toml cli tool is not installed on everybody's computer, while `sed` is rather commonly installed. So personally I lean towards leaving sed or use another commonly used tool in this case.

Reply:
These tools also depend on `jq`, a JSON tool, which isn't exactly standard :) Same for Docker. If these scripts are going to run inside Docker, it's fairly simple to ensure the environment has whatever is needed. If it's being run from outside Docker, you could move the parsing logic into Docker and let Docker handle the dependencies.

Reply:
Good points. Personally I lean towards just keeping `sed` as it was the tool used before and currently I think it's still fine. If we have more manipulation in the toml files we should introduce a toml cli tools like https://github.com/TomWright/dasel or https://github.com/kislyuk/yq